spyder/plugins/workingdirectory/plugin.py

```diff
@@ -165,7 +164,7 @@ def on_projects_teardown(self):
 
     # --- Public API
     # ------------------------------------------------------------------------
-    def chdir(self, directory: str, sender_plugin: Optional[str] = None):
+    def chdir(self, directory: str, sender_plugin: Optional[str] = None, server_id: Optional[str] = None):
```

Comment:
```suggestion
    def chdir(
        self,
        directory: str,
        sender_plugin: Optional[str] = None,
        server_id: Optional[str] = None
    ):
```

- For 79 columns.
- Please document the new kwarg in our docstring.
- Add a regular entry in our changelog about the new kwarg.